src/EPR.RegulatorService.Facade.Core/Clients/PrnBackendServiceClient/PrnBackendServiceClient.cs

```diff
@@ -0,0 +1,15 @@
+﻿using EPR.RegulatorService.Facade.Core.Configs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Diagnostics.CodeAnalysis;
+namespace EPR.RegulatorService.Facade.Core.Clients.PrnBackendServiceClient;
```

Comment:
Minor again, but could add a line break:
```suggestion
using System.Diagnostics.CodeAnalysis;

namespace EPR.RegulatorService.Facade.Core.Clients.PrnBackendServiceClient;
```